packages/flutter/test_fixes/material/theme_data.dart

```diff
@@ -237,4 +237,11 @@ void main() {
 
   // Changes made in https://github.com/flutter/flutter/pull/131455
   ThemeData themeData = ThemeData.copyWith(useMaterial3: false);
+
+  // Changes made in https://github.com/flutter/flutter/pull/155072
+  ThemeData themeData = ThemeData();
+  themeData = ThemeData(dialogBackgroundColor: Colors.orange);
```

Comment:
What happens if ThemeData.dialogTheme is already provided? It's been a while since I wrote one of these. :)

Reply:
Good question! I'll test and see what happens. Update the fix :)

Reply:
Updated to fix to handle ` ThemeData.dialogTheme` already there.